File: Android Learning/Android in Java/BundlePassing/app/src/main/java/com/example/bundlepassing/MainActivity.java
package com.example.bundlepassing;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity{
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnNext = findViewById(R.id.btnNext);

        Intent iNext;
        iNext = new Intent(MainActivity.this, SecondActivity.class);
        iNext.putExtra("name", "MS");
        iNext.putExtra("roll_no", "3");
        iNext.putExtra("title", "Student Info");

        btnNext.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(iNext);
            }
        });


    }
}

File: MyFirstApp/app/src/main/java/com/example/myfirstapp/MainActivity.kt
package com.example.myfirstapp

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.example.myfirstapp.ui.theme.MyFirstAppTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) { //to save content in the same page
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //val and var
        val buttonSkills = findViewById<Button>(R.id.btnSkills)
        val buttonEdu = findViewById<Button>(R.id.btnEdu)
        val buttonWork = findViewById<Button>(R.id.btnWork)
        val buttonAcheive = findViewById<Button>(R.id.btnAchieve)

        buttonSkills.setOnClickListener {
            val intent = Intent(this, SkillsActivity::class.java)
            startActivity(intent)
        }

        buttonEdu.setOnClickListener {
            val intent = Intent(this, EduActivity::class.java)
            startActivity(intent)
        }

        buttonWork.setOnClickListener {
            val intent = Intent(this, WorkActivity::class.java)
            startActivity(intent)
        }

        buttonAcheive.setOnClickListener {
            val intentAchieve = Intent(this, AchieveActivity::class.java)
            startActivity(intentAchieve)
        }
    }
}




